modules/gh/gh.go

```diff
@@ -0,0 +1,304 @@
+}
+
+func StartGroup(ctx context.Context, args ...object.Object) object.Object {
+	if err := arg.Require("gh.start_group", 1, args); err != nil {
+		return err
+	}
+	message := printableValue(args[0])
+	stdout := os.GetDefaultOS(ctx).Stdout()
+	return runWorkflowCommand(stdout, "group", message, nil)
+}
+
+func EndGroup(ctx context.Context, args ...object.Object) object.Object {
+	if err := arg.Require("gh.end_group", 0, args); err != nil {
+		return err
+	}
+	stdout := os.GetDefaultOS(ctx).Stdout()
+	return runWorkflowCommand(stdout, "endgroup", "", nil)
+}
+
+func SetOutput(ctx context.Context, args ...object.Object) object.Object {
+	if err := arg.Require("gh.set_output", 2, args); err != nil {
+		return err
+	}
+	key := printableValue(args[0])
+	value := printableValue(args[1])
+
+	risorOS := os.GetDefaultOS(ctx)
+	outputFile := risorOS.Getenv("GITHUB_OUTPUT")
+	if outputFile != "" {
+		return appendWorkflowFile(outputFile, workflowFileKeyValue(key, value))
+	}
+
+	stdout := risorOS.Stdout()
+	// Using "::set-output::" command is deprecated, but it's a good enough fallback
+	return runWorkflowCommand(stdout, "set-output", value, map[string]any{"name": key})
+}
+
+func SetEnv(ctx context.Context, args ...object.Object) object.Object {
+	if err := arg.Require("gh.set_env", 2, args); err != nil {
+		return err
+	}
+	key := fmt.Sprint(printableValue(args[0]))
+	value := fmt.Sprint(printableValue(args[1]))
+
+	risorOS := os.GetDefaultOS(ctx)
+	risorOS.Setenv(key, value)
+
+	envFile := risorOS.Getenv("GITHUB_ENV")
+	if envFile != "" {
+		return appendWorkflowFile(envFile, workflowFileKeyValue(key, value))
+	}
+
+	stdout := risorOS.Stdout()
+	// Using "::set-env::" command is deprecated, but it's a good enough fallback
+	return runWorkflowCommand(stdout, "set-env", value, map[string]any{"name": key})
+}
+
+func AddPath(ctx context.Context, args ...object.Object) object.Object {
+	if err := arg.Require("gh.add_path", 1, args); err != nil {
+		return err
+	}
+	path := printableValue(args[0])
+	pathStr := fmt.Sprint(path)
+
+	risorOS := os.GetDefaultOS(ctx)
+	oldPath := risorOS.Getenv("PATH")
+	risorOS.Setenv("PATH", fmt.Sprintf("%s%c%s", pathStr, goos.PathListSeparator, oldPath))
+
+	pathFile := risorOS.Getenv("GITHUB_PATH")
+	if pathFile != "" {
+		return appendWorkflowFile(pathFile, pathStr)
+	}
+
+	stdout := risorOS.Stdout()
+	// Using "::add-path::" command is deprecated, but it's a good enough fallback
+	return runWorkflowCommand(stdout, "add-path", path, nil)
+}
+
+func Module() *object.Module {
+	return object.NewBuiltinsModule("gh", map[string]object.Object{
+		"is_debug":    object.NewBuiltin("gh.is_debug", IsDebug),
+		"log_info":    object.NewBuiltin("gh.log_info", LogInfo),
+		"log_debug":   object.NewBuiltin("gh.log_debug", LogDebug),
+		"log_notice":  object.NewBuiltin("gh.log_notice", LogNotice),
+		"log_warning": object.NewBuiltin("gh.log_warning", LogWarning),
+		"log_error":   object.NewBuiltin("gh.log_error", LogError),
```

Comment:
Are all these needed? We can check env vars and print messages with `risor` already....

Reply:
The logging ones are helper functions, as handling multiline messages and the file parameters can be quite tedious when they're based on variables.

The `log_info` I could agree we can remove, as it's practically just an alias for `println`. 

Regarding `is_debug` it exposes the API in a nicer way. But I'm OK with removing it too, if you think it bloats the API.

The remaining logger functions I really think should stay though.

Reply:
I've removed `log_info`

Reply:
Oh and I forgot to mention, these functions were practically ported from here: https://github.com/actions/toolkit/blob/1fe633e27c4cc74616d675b71163f59f9d084381/packages/core/src/core.ts#L297-L299

That's why I added the `log_info` to begin with.